src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -209,15 +209,39 @@ private void setUnsavedFieldsOnCallback(Callback<SscsCaseData> callback) {
             if (isNotBlank(processingVenue)) {
                 callback.getCaseDetails().getCaseData().setProcessingVenue(processingVenue);
             }
+            setWorkAllocationCategories(appeal, callback);
+        } else {
+            setUnknownCategory(callback);
+        }
 
-            if (workAllocationFeature) {
-                Optional<Benefit> benefit = Benefit.getBenefitOptionalByCode(appeal.getBenefitType().getCode());
-                if (benefit.isPresent()) {
-                    callback.getCaseDetails().getCaseData().getWorkAllocationFields().setCategorys(benefit.get());
-                }
+        setWorkallocationFields(appeal, callback);
+    }
+
+    private void setWorkAllocationCategories(Appeal appeal, Callback<SscsCaseData> callback) {
+        if (workAllocationFeature) {
+            Optional<Benefit> benefit = Benefit.getBenefitOptionalByCode(appeal.getBenefitType().getCode());
+            if (benefit.isPresent()) {
+                callback.getCaseDetails().getCaseData().getWorkAllocationFields().setCategories(benefit.get());
+            }
+        }
+    }
+
+    private void setUnknownCategory(Callback<SscsCaseData> callback) {
+        FormType formType = callback.getCaseDetails().getCaseData().getFormType();
+        if (formType != null) {
+            if (formType.equals(FormType.SSCS5)) {
+                DynamicListItem caseManagementCategoryItem = new DynamicListItem("sscs5Unknown", "SSCS5 Unknown");
+                List<DynamicListItem> listItems = Arrays.asList(caseManagementCategoryItem);
+                callback.getCaseDetails().getCaseData().getWorkAllocationFields().setCaseManagementCategory(new DynamicList(caseManagementCategoryItem, listItems));
+            } else {
+                DynamicListItem caseManagementCategoryItem = new DynamicListItem("sscs12Unknown", "SSCS1/2 Unknown");
+                List<DynamicListItem> listItems = Arrays.asList(caseManagementCategoryItem);
+                callback.getCaseDetails().getCaseData().getWorkAllocationFields().setCaseManagementCategory(new DynamicList(caseManagementCategoryItem, listItems));
             }
         }
+    }
 
+    private void setWorkallocationFields(Appeal appeal, Callback<SscsCaseData> callback) {
```

Comment:
I know you not changed as part of this PR but it went in the other day... Just reviewing this and think there is an edge case here. They could theoretically create a case directly in ExUI without going through Bulk Scan. Assuming it is incomplete, when when they trigger the 'Validate appeal' event then the logic on line 251 might cause that ogdType to be empty, which will then mess up the UI. 

I actually had a similar problem in a PR I did the other day here: https://github.com/hmcts/sscs-track-your-appeal-notifications/pull/1989/files#diff-bbc4ee9b605974afeba6cfaf26480af787bb05f7946dad2b6c198d56783e235aR287

Wondering if you can do something similar here e.g. if you can't find the 'formType' then use the benefit type to work out what the ogdType should be?